apm-agent-plugins/apm-ratpack-plugin/src/main/java/co/elastic/apm/agent/ratpack/AbstractRatpackInstrumentation.java

```diff
@@ -0,0 +1,39 @@
+/*-
+ * #%L
+ * Elastic APM Java agent
+ * %%
+ * Copyright (C) 2018 - 2019 Elastic and contributors
+ * %%
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ * #L%
+ */
+package co.elastic.apm.agent.ratpack;
+
+import co.elastic.apm.agent.bci.ElasticApmInstrumentation;
+
+import java.util.Collection;
+import java.util.Collections;
+
+public abstract class AbstractRatpackInstrumentation extends ElasticApmInstrumentation {
+
```

Comment:
Add a classloader matcher so that the instrumentation only gets applied for applications which actually have Ratpack on the classpath.

Example:
https://github.com/elastic/apm-agent-java/blob/ed3480abd5de58c9d12fc2500b1b06389c1dafec/apm-agent-plugins/apm-asynchttpclient-plugin/src/main/java/co/elastic/apm/agent/asynchttpclient/AbstractAsyncHttpClientInstrumentation.java#L65-L68